static/css/v3/heros.css

```
@@ -3,602 +3,826 @@
+  background-color: #081040;
+}
 
 .dark .hero-home--with-background-image {
- background-color: transparent;
+  background-color: transparent;
 }
 
- .hero-home--no-image {
+.hero-home--no-image {
   background-image: none;
   background: none;
```

Stylelint (17.12.0)

[error] Overridden property "background-image" by shorthand "background" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)